### PR TITLE
[Tab] Fix inconsistent border-radius 

### PR DESCRIPTION
**Pull request by gwyneplaine**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/polaris-summer-editions/issues/434

### WHAT is this pull request doing?
Before: 
Screenshot 2023-06-27 at 11 57 18 am

After: 
Screenshot 2023-06-27 at 11 56 05 am

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

**Review by aveline — APPROVED** [review]
We shouldn't include a change log for se23 only changes. Add a `Skip Changelog` label to the PR for CI to pass. Otherwise looks good!

**Review by sarahill — APPROVED** [review]

File: .changeset/three-oranges-eat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': patch
+---
+
+[Tabs] fix inconsistent DisclosureActivator border-radius
```
Comment (aaronccasanova) [inline .changeset/three-oranges-eat.md]: In addition to @aveline's callout, we generally don't prefix component names in square brackets in changelogs. I can't quickly find the original documentation, but I faintly remember reading docs stating to use past tense and wrap component/prop names in `` `back ticks` ``

**Review by aaronccasanova — APPROVED** [review]

**Pull request merged**